README.md

```diff
@@ -186,6 +187,7 @@ url = "index.xml"
 | `serverfault`   | serverfault.com                 |
 | `soundcloud`    | soundcloud.com                  |
 | `shutterstock`  | shutterstock.com                |
+| `signal`        | signal.me                       |
```

Comment:
The official website for signal is signal.org . Could you please change this?

Reply:
Good catch! Yeah, I updated that in my latest push just now.